### PR TITLE
Update versions specified in README.md

### PR DESCRIPTION
**Pull request by jtx1999**
On the latest STARTS version, we are supporting Java 8-15, and multiple OS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,10 +8,10 @@ for Maven-based Java programs.
...
 ## Prerequisites
 
-1. Java 1.8
+1. Java 8 - 15
 2. Maven 3.2.5 and above
 3. Maven Surefire 2.14 and above
-4. Operating System: Linux or OSX
+4. Operating System: Linux, OSX or Windows
 
 ## Integrating STARTS Plugin from Maven Central
```
Comment (owolabileg) [inline README.md:16]: we only have 1.3 on maven central at this time. What you want to do is to add revert the change below, then add another set of instructions for running the latest development version.

Comment (jtx1999) [reply README.md]: Updated

File: README.md [hunk 2]
```diff
@@ -31,10 +31,26 @@ Change the pom.xml to add the configuration for the STARTS plugin:
 
 ## Building STARTS from source
 
+Build from source to use the latest development version, which supports Java 8-15, on Linux, OSX, or Windows.
+
 1. `git clone https://github.com/TestingResearchIllinois/starts`
 2. `cd starts`
 3. `mvn install`
 
+Then change the pom.xml to add the configuration for the latest version of STARTS plugin:
```
Comment (owolabileg) [inline README.md:40]: Then --> Then,

latest version of --> latest development version of the

Comment (jtx1999) [reply README.md]: Updated
